hail/python/test/hailtop/batch/test_batch.py

```diff
@@ -88,6 +88,52 @@ def test_single_job(self):
 
             assert self.read(output_file.name) == msg
 
+    def test_single_job_with_entrypoint_no_image(self):
+        with tempfile.NamedTemporaryFile('w') as output_file:
+            msg = 'hello world'
+
+            b = self.batch()
+            j = b.new_job()
+            j.entrypoint('echo foobar')
+            j.command(f'echo "{msg}" > {j.ofile}')
+
+            b.write_output(j.ofile, output_file.name)
+            b.run()
+            out = self.read(output_file.name)
+
+            assert self.read(output_file.name) == msg
+
+    def test_single_job_with_entrypoint(self):
```

Comment:
This test isn't working in CI (file not found), works fine on local. @danking as I don't have access to the cluster anymore, would you have any suggestions?